jetty-core/jetty-http2/jetty-http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Stream.java

```diff
@@ -1053,4 +1058,9 @@ public List<StreamFrame> getFrames()
             return frames;
         }
     }
+
+    public interface DispatchableListener extends Stream.Listener
```

Comment:
This interface should have javadoc explaining it's needed for backward compatibility reasons.